instana/instrumentation/aiohttp/server.py

```diff
@@ -49,6 +49,14 @@ async def stan_middleware(request, handler):
                 response.headers['Server-Timing'] = "intid;desc=%s" % scope.span.context.trace_id
 
             return response
+
+        except aiohttp.web_exceptions.HTTPClientError as e:
```

Comment:
This one would just handle the 4XX responses though, not e.g. 502. `HTTPException` is the right exception class to catch here.

Reply:
Ah yes, i was only focusing on the 4xx. But probably general HTTPException would be better

Reply:
ah wait no, HTTPSuccessful is also under HTTPException apparently.  How about HTTPError? this will capture all 4xx and 5xx without 2xx and 3xx

Reply:
Let's move this conversation elsewhere. I see other things that don't seem to be working as intended.